### PR TITLE
zvbi: 0.2.42-unstable-2024-03-21 -> 0.2.42-unstable-2024-11-29

### PR DESCRIPTION
**Pull request by alyssais**
## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by Atemu** [comment]
@ofBorg build pkgsMusl.zvbi pkgsMusl.ffmpeg-headless pkgsStatic.zvbi pkgsStatic.ffmpeg-headless

**Review by Atemu — APPROVED** [review]
Not my field of expertise but the diff and patch LGTM.

**Review by jopejoe1 — COMMENTED** [review]
We could also just update the package to the newest commit, as we are already using an unstable commit, and there is only one other code changing commit in between.

File: pkgs/by-name/zv/zvbi/package.nix [hunk 1]
```diff
@@ -20,6 +21,14 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-1VTTNfXZ12hJWiW+M1WsE8H/PejrJsT/E2RHZ3OSIC8=";
   };
 
+  patches = [
+    (fetchpatch {
```
[the remark this answers is not in the document]
Comment (alyssais) [reply pkgs/by-name/zv/zvbi/package.nix]: Changed to an update, so no patch. :)

**Comment by alyssais** [comment]
Good idea! Done.

File: pkgs/by-name/zv/zvbi/package.nix (hunk 1 above)
[the remark this answers is not in the document]
Comment (Atemu) [reply pkgs/by-name/zv/zvbi/package.nix]: Neat :)

**Comment by Atemu** [comment]
Ah, it's probably good you cancelled those ofBorg builds, I didn't realise this targets staging ^^'

**Review by jopejoe1 — APPROVED** [review]

**Pull request merged**